service/docs/redis.md

```diff
@@ -3,23 +3,24 @@
 BOPTEST-Service currently uses Redis for two purposes.
 
 1. Storing test metadata.
-2. A pub/sub message bus to communicate with workers. 
+2. A pub/sub message bus to communicate with workers.
 
 ## Test Metadata:
 
 A hash under the key `tests:${testid}` is used to store test metadata.
 
-| Key                                      | Field       | Value
-| ---------------------------------------- | ----------- | -----------------------------
-| `tests:${testid}`                        | `status`    | `Queued \| Running`
-| `tests:${testid}`                        | `timestamp` | Epoch seconds when test is queued
-| `tests:${testid}`                        | `user`      | The OAuth user "sub", or undefined for anonymous tests
+| Key               | Field       | Value                                                                                    |
+| ----------------- | ----------- | ---------------------------------------------------------------------------------------- |
+| `tests:${testid}` | `status`    | `Queued \| Running`                                                                      |
+| `tests:${testid}` | `timestamp` | Epoch seconds when test is queued.                                                       |
+| `tests:${testid}` | `user`      | The OAuth user "sub", or undefined for anonymous tests.                                  |
+| `tests:${testid}` | `host`      | The value of HOSTNAME for the worker running the test. On K8s this will be the pod name. |
```

Comment:
There are a number of diffs here, due to formatting adjustment, but the addition of `host` is the actual change. For an instance deployed on Kubernetes, host will be the worker pod name. This is really useful for connecting to a pod that may be running a test and doing diagnostics if there is an issue.